File: proyectos/java/CursoPOOUber/Java/Car.java
public class Car {
    Integer id;
    String licence;
    Account driver;
    Integer passenger;

    public Car(String licence, Account driver){
        this.licence = licence;
        this.driver = driver;
    }

    @Override
    public String toString() {
        return "Car [id=" + id + ", licence=" + licence + ", driver=" + driver + ", passenger=" + passenger + "]";
    }
    
}

File: proyectos/java/CursoPOOUber/JS/Car.js
class Car {
  constructor(licence, driver) {
    this.id;
    this.licence = licence;
    this.driver = driver;
    this.passenger;
  }

  printDataCar() {
    console.log(this.driver);
    console.log(this.driver.name);
    console.log(this.driver.document);
  }
}
